packages/kit/src/exports/public.d.ts

```diff
@@ -25,6 +25,7 @@ import {
 	LayoutParams as AppLayoutParams,
 	ResolvedPathname
 } from '$app/types';
+// @ts-ignore - some users might not have this installed.
```

Comment:
I don't think will fix the issue. `dts-buddy`, which we use to generate our published types, doesn't preserve these TypeScript escape hatch comments. The TypeScript compiler also doesn't preserve these comments. We used to do this for the app generated types but we had to stop doing it. The eventual solution was https://github.com/sveltejs/kit/pull/14207

Reply:
See my comment on the main discussion.